scenario/scenario.go

```diff
@@ -49,21 +51,20 @@ func CreateZRXForWETHSignedTestOrder(t *testing.T, makerAddress, takerAddress co
 
 	ganacheAddresses := ethereum.NetworkIDToContractAddresses[constants.TestNetworkID]
 
-	ethClient, err := ethclient.Dial(constants.GanacheEndpoint)
-	require.NoError(t, err)
-
 	weth9, err := wrappers.NewWETH9(ganacheAddresses.WETH9, ethClient)
 	require.NoError(t, err)
 
 	// Convert ETH-WETH
 	opts := &bind.TransactOpts{
 		From:   takerAddress,
 		Value:  wethAmount,
-		Signer: getTestSignerFn(takerAddress),
+		Signer: GetTestSignerFn(takerAddress),
 	}
 	txn, err := weth9.Deposit(opts)
 	require.NoError(t, err)
-	receipt, err := bind.WaitMined(context.Background(), ethClient, txn)
+	ctx, cancelFn := context.WithTimeout(context.Background(), 4*time.Second)
```

Comment:
This pattern for calling `WaitMined` also shows up a lot and could potentially be refactored to a function.

Reply:
Great idea! Added `waitTxnSuccessfullyMined`